android/src/main/java/com/thunderhead/one_sdk_flutter/OnePlugin.java

```diff
@@ -24,15 +24,35 @@
 import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
 import io.flutter.plugin.common.MethodChannel.Result;
 import io.flutter.plugin.common.PluginRegistry.Registrar;
```

Comment:
I thought we are dropping this and migrating to the new API?

Reply:
We are keeping the registerWith() method to remain compatible with apps that don’t use the v2 Android embedding